ods2md.py

```diff
@@ -33,14 +31,16 @@
     'W': 2,
 }
 
+LINEFEED_REGEX=re.compile('(\r\n|\r|\n)', flags=re.MULTILINE)
+
 def display_text(cell):
     v = cell.value
     if isinstance(v, float):
-        return '{:g}'.format(v)
+        return LINEFEED_REGEX.sub('<br>', '{:g}'.format(v))
```

Comment:
```suggestion
        return '{:g}'.format(v)
```

why would there be new lines in floats 🤔

Reply:
Late night programming? LOL